yt_dlp/networking/_websockets.py

```diff
@@ -5,20 +5,26 @@
 import ssl
 import sys
 
-from ._helper import create_connection, select_proxy, make_socks_proxy_opts, create_socks_proxy_socket
-from .common import Response, register_rh, Features
+from ._helper import (
+    create_connection,
+    create_socks_proxy_socket,
+    make_socks_proxy_opts,
+    select_proxy,
+)
+from .common import Features, Response, register_rh
```

Comment:
unrelated change

Reply:
We'll need to move this to a cleanup commit